### PR TITLE
Remove aptitude from Ansible playbooks

### PR DESCRIPTION
**Pull request by pierrepo**
Enforce ansible >= 2.9 in doc

- [x] Add / update the documentation

### CODE REVIEW

**Comment by pierrepo** [comment]
This should fix #149.

File: docs/install/ansible.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ Make sure `Ansible <https://docs.ansible.com/ansible/latest/index.html>`_ is ins
 
 .. code-block:: bash
 
-    python -m pip install ansible
+    python -m pip install ansible>=2.9
```
Comment (jtpio) [inline docs/install/ansible.rst:47]: Maybe we should briefly explain why the version should be `>=2.9`?

Comment (pierrepo) [reply docs/install/ansible.rst:47]: That seems reasonable.

I added a note on this topic.

File: docs/install/ansible.rst [hunk 2]
```diff
@@ -44,7 +44,12 @@ Make sure `Ansible <https://docs.ansible.com/ansible/latest/index.html>`_ is ins
...
+
+.. note::
+
+    We recommend ansible >= 2.9 to get ride of unecessary warning message 
+    regarding the use of aptitude.
```
Comment (jtpio) [inline docs/install/ansible.rst:52]:
```suggestion
    We recommend ``ansible>=2.9`` to discard the warning messages 
    regarding the use of ``aptitude``.
```

**Comment by jtpio** [comment]
Thanks!

**Comment by pierrepo** [comment]
:tada:

**Pull request merged**